upgrading/upgrade-operator.adoc

```diff
@@ -9,15 +9,24 @@ toc::[]
 [role="_abstract"]
 Upgrades through the {rh-rhacs-first} Operator are performed automatically or manually, depending on the *Update approval* option you chose at installation.
 
-{product-title-short} 4.0 includes a significant architectural change, moving Central’s database to PostgreSQL. Because of this change, {product-title-short} 4.0 Operator is published by a new subscription channel. Therefore, as part of the upgrade instructions, you must manually change the subscription channel to upgrade from {product-title-short} 3.74 to {product-title-short} 4.0.
+To get upgrade instructions, see the documentation for your product version.
 
-//If you installed {product-title-short} using the Operator and selected *Automatic* in the *Update approval* field, {product-title-short} is automatically updated when a new software version is released. If you selected *Manual*, you must approve subsequent Operator updates by using Operator Lifecycle Manager (OLM). For more information, see link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html/operators/administrator-tasks#olm-approving-pending-upgrade_olm-upgrading-operators[Manually approving a pending Operator update].
+* For upgrading from version 3.74 to 4.x, see the following resources:
+** link:https://docs.openshift.com/acs/3.74/upgrading/upgrade-operator.html[Upgrading by using the Operator]
+** link:https://docs.openshift.com/acs/3.74/upgrading/upgrade-helm.html [https://docs.openshift.com/acs/3.74/upgrading/upgrade-helm.html]
+** link:https://docs.openshift.com/acs/3.74/upgrading/upgrade-roxctl.html[Manually Manually upgrading using the roxctl CLI]
```

Comment:
1. Would having links to docs.openshift.com/acs/* constitute a problem for this documentation when it's rendered under access.redhat.com (i.e. <https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_security_for_kubernetes/4.2/html/upgrading/upgrade-operator>)?
It feels a bit odd that links will take users outside of access.redhat.com.
2. Something's a bit broken with the Helm link, see the screenshot.
![image](https://github.com/openshift/openshift-docs/assets/537715/b425a836-35e2-4899-922d-68c369f85e91)

Reply:
Yeah, still fixing links and stuff. But good point - I'll change the links to the Customer Portal since that's the preferred way. (Just users accessing from docs.openshift.com will have the issue that they get taken to the Customer Portal.) Unfortunately there is no way to use relative links to a different version when you are viewing a specific version - i.e., if I use a relative link in 4.0, it will always go to the 4.0 version of the doc.